### PR TITLE
Create serializers with validators 

### PR DESCRIPTION
**Pull request by thiagogquinto**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

This *Pull Request* implements serializers with validators for all the models already defined. 

### Related issue

- #295

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks for this, @thiagogquinto! I'm a bit busy today and the next few days, but we'll check this for you and provide some feedback 😊

CC @momanyisamuel and @kurtisolo

**Comment by kurtisolo** [comment]
In my opinion, it looks good; I haven't tested it, but I don't see any obvious issues. The addition of utils also makes sense to me. Maybe it could be done differently, but for now, it seems fine. Perhaps I would suggest using:
```
 class Meta:
     model = EventAttendee
     fields = '__all__'
```
for each class in serializers.py to reduce the amount of code and make it more readable?

**Comment by thiagogquinto** [comment]
Thanks @kurtisolo, you are absolutely right, this way looks much better.

File: backend/authentication/serializers.py [hunk 1]
```diff
@@ -1,23 +1,89 @@
...
+            )
+        return data
+class SupportSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Support
+        fields = '__all__'
+
+    def validate(self, data):
+        
+        if data["supporter_type"] == data["supported_type"]:
```
Comment (andrewtavis) [inline backend/authentication/serializers.py:25]: This one isn't needed as they can be the same if say a user is supporting another user. The entity cannot be the same for sure, but then for this we need to keep in mind that we need to not repeat ids for organizations for those of users :)

I can remove this. No stress! 😎

**Comment by andrewtavis** [comment]
Basic question on my part: is it standard that validation errors are incomplete sentences? I'd assume that these should follow Python full-line comment standards with capitalization and a period at the end (inline is lower case and no period as is present). Will merge regardless, but in case there's an explicit preference in the Django community I'll likely go through and change these :)

File: backend/entities/serializers.py [hunk 2]
```diff
@@ -1,18 +1,189 @@
...
+
+        return data
+
+
+class OrganizationMemberSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = OrganizationMember
+        fields = "__all__"
+
+    def validate(self, data):
```
Comment (andrewtavis) [inline backend/entities/serializers.py:68]: This one here is also not necessary. Would be great to get feedback on the various roles we'll include for orgs, but generally we just have these three right now as special roles with more permissions :)

**Review by andrewtavis — APPROVED** [review]
Sorry for a bit of a wait on this review, @thiagogquinto! Really appreciate all the care that you've put into this PR, and hope to have more contributions from you in the future given your knowledge of what we're building in the backend now! 🚀 I'll go through now and remove the two mentioned validations that aren't necessary. This is really great! 😊

**Pull request merged**